### ISSUE TITLE
One should be able to leave a group

### ISSUE THREAD

**Issue by fmorency** [liftedinit/manifest-app#50]
A user should be able to leave a group without passing by a proposal.

The "Leave Group" message type should be removed from the proposal creation UI.

**Comment by chalabi2** [comment]
I believe the only way to leave a group is through a proposal

**Comment by fmorency** [comment]
> I believe the only way to leave a group is through a proposal

I was able to leave a group by calling

```
manifestd tx group leave-group manifest1hj5fveer5cjtn4wd6wstzugjfdxzl0xp8ws9ct 2 \
  --from manifest1hj5fveer5cjtn4wd6wstzugjfdxzl0xp8ws9ct \
  --node https://nodes.liftedinit.tech/manifest/testnet/rpc \
  --gas auto \
  --gas-prices 0.0011umfx \
  --gas-adjustment 1.4 \
  --chain-id manifest-ledger-testnet
```

No proposal required.

**Comment by chalabi2** [comment]
fixed in #88

**Comment by chalabi2** [comment]
fixed in #88
